test/testapp/compile.js

```diff
@@ -19,11 +21,64 @@ qx.Class.define("qxl.compilertests.testapp.LibraryApi", {
   extend: qx.tool.cli.api.LibraryApi,
   
   members: {
+    __startedSassWatch: false,
+    
     async load() {
       let command = this.getCompilerApi().getCommand();
+      command.addListener("making", () => this._onMaking());
       command.addListener("checkEnvironment", e => this._appCompiling(e.getData().application, e.getData().environment));
     },
     
+    _onMaking() {
+      debugger;
```

Comment:
Is this here on purpose? I haven't used this in a while, but somehow remember it would stop execution...

Reply:
it only stops (pauses) under the debugger, but it shouldnt be there - removing it now